apps/src/sites/studio/pages/devise/registrations/edit.js

```diff
@@ -61,8 +91,6 @@ $(document).ready(() => {
     if (needToConfirmEmail) {
       e.preventDefault();
       showConfirmEmailModal(onCancelModal, onSubmitModal(e));
-    } else if ($('#user_email').length) {
-      prepareEmailData();
```

Comment:
Are we sure that everything else works fine if this is removed? I don't really remember the logic here, and when `prepareEmailData()` needs to be called or not.

Reply:
I presume this is ok because the email fields have been removed from the form, and are now handled by the React modal.

Reply:
Andrew's got it: I took a pretty close look at this.  I've removed the only `#user_email` element on this page to replace it with the uneditable email display and "Update" link, so `$('#user_email').length` would always evaluate false, making this dead code.